README.md

```diff
@@ -17,7 +17,7 @@ To learn how to contribute to the MetaMask project itself, visit our [Internal D
 - Install [Node.js](https://nodejs.org) version 16
     - If you are using [nvm](https://github.com/nvm-sh/nvm#installing-and-updating) (recommended) running `nvm use` will automatically choose the right node version for you.
 - Install [Yarn](https://yarnpkg.com/en/docs/install)
```

Comment:
This install link is for Yarn v1. Should probably be explicit about needing v3 with the text as well:

```suggestion
- Install [Yarn v3](https://yarnpkg.com/getting-started/install)
```